From ca8d51e6d0f78cc170cc212dd7ef6664d2e662f4 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Fri, 20 May 2016 09:16:35 -0400
Subject: Revert "scrolledwindow: Remove child before destroying self"

This reverts commit dcb4b48b29559ca632bb020a3b8eb2b9188b02e7.

This was causing crashes in the filechooser in some applications.

https://bugzilla.gnome.org/show_bug.cgi?id=766694
---
 gtk/gtkscrolledwindow.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/gtk/gtkscrolledwindow.c b/gtk/gtkscrolledwindow.c
index 0918d72..1b52082 100644
--- a/gtk/gtkscrolledwindow.c
+++ b/gtk/gtkscrolledwindow.c
@@ -2672,11 +2672,6 @@ gtk_scrolled_window_destroy (GtkWidget *widget)
 {
   GtkScrolledWindow *scrolled_window = GTK_SCROLLED_WINDOW (widget);
   GtkScrolledWindowPrivate *priv = scrolled_window->priv;
-  GtkWidget *child;
-
-  child = gtk_bin_get_child (GTK_BIN (widget));
-  if (child)
-    gtk_container_remove (GTK_CONTAINER (widget), child);
 
   remove_indicator (scrolled_window, &priv->hindicator);
   remove_indicator (scrolled_window, &priv->vindicator);
-- 
cgit v0.12

